amr-wind/equation_systems/vof/volume_fractions.H

```diff
@@ -493,10 +493,11 @@ AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE bool interface_band(
     const int j,
     const int k,
     amrex::Array4<amrex::Real const> const& volfrac,
-    const int n_band = 1) noexcept
+    const int n_band = 1,
+    const amrex::Real tiny = 1e-12) noexcept
 {
     // n_band must be <= number of vof ghost cells (3)
```

Comment:
Just some small questions: Do you want to make this check explicit in the places where you call this function? Also, I don't see you using `tiny` as an argument so do you want to consider leaving it as an variable inside this function as it was?

Reply:
good questions. I think it would be a lot of scattered checks if I did make it explicit, and I don't think I have enough information inside the function to do the check. I don't think it's a big enough concern to add checks.

I found this bug while I was developing something else, which was using tiny as an argument. I'll remove that from this PR and add it if I use it in another PR.